src/app/layouts/components/header/header.component.scss

```
@@ -39,11 +38,41 @@
     }
 
     .avatar {
-        width: 60px;
-        height: 60px;
+        width: 40px;
+        height: 40px;
     }
 
     .menu-sesion {
         cursor: pointer;
     }
 }
+
+@include from-breakpoint($break-medium) {
+    .header {
+        font-size: $font-medium;
+
+        .logo-img {
+            @include screen-logo-tablet ();
```

remove space after parentheses
```suggestion
            @include screen-logo-tablet();
```